src/firefly/java/edu/caltech/ipac/visualize/net/IrsaImageParams.java

```diff
@@ -37,6 +37,9 @@ public String getUniqueString() {
              case IRIS :
                        retval= "iris-" + super.toString() + _band + _size;
                        break;
+             case ATLAS:
+                 retval = "atlas-"+ super.toString();
```

Comment:
**Really Important!**

i don't think string is unique.  It is a cache key and a file name in the working directory.

I think `AtlasParams` needs override `getUniqueString`. needs to user

some think like:
```
return "atlas-" +_band+"-"+xtraFilter+"-"+_instrument +"-"+String.format(Locale.US,"%8.6f%8.6f",_ra,_dec)
```

Reply:
I know, i discover this logic while implementing, the unique string is used heavily.
But i overwrite it, is less apparent but see AtlasImageParams.java. I've added it there.

Reply:
looks better, as long is it produces a reproducible  unique string.